evalml/tests/component_tests/test_baseline_classifier.py

```diff
@@ -78,14 +78,14 @@ def test_baseline_binary_random_weighted(X_y_binary):
     clf.fit(X, y)
 
     assert clf.classes_ == [0, 1]
-    expected_predictions = pd.Series(get_random_state(0).choice(np.unique(y), len(X), p=percent_freq), dtype="Int64")
+    expected_predictions = pd.Series(get_random_state(0).choice(np.unique(y), len(X), p=percent_freq), dtype="int64")
```

Comment:
Curious, is it necessary to specify `dtype="int64"`? :O Does it not default to this? (Previously it didn't default to Int64 hence the forced conversion but maybe we don't need these for the tests anymore?

(Also okay if left in for clarity heh)